feature/profile/src/commonMain/kotlin/io/github/confsched/profile/components/FlippableProfileCard.kt

```diff
@@ -92,47 +88,51 @@ private fun ProfileCard(
     interactionsEnabled: Boolean,
     modifier: Modifier = Modifier,
 ) {
-    Card(
-        modifier = modifier
-            .clickable(
-                enabled = interactionsEnabled,
-                interactionSource = remember { MutableInteractionSource() },
-                indication = null,
-            ) {
-                onFlippedChange(!isFlipped)
+    WithDeviceOrientation {
```

Comment:
This is more of a personal preference, but since `DeviceOrientationScope` doesn’t really do anything special besides holding `orientation`, maybe we could try providing `orientation` directly as a lambda argument instead…?

Reply:
Alternatively, it could be interesting to make the tiltEffect and hologramaticEffect modifiers require DeviceOrientationScope as a context parameter.

That said, it might be a bit harder to work with, so personally I’d recommend passing orientation directly as a lamda argument.